zipline/data/minute_bars.py

```diff
@@ -898,8 +899,22 @@ class BcolzMinuteBarReader(MinuteBarReader):
     zipline.data.minute_bars.BcolzMinuteBarWriter
     """
     FIELDS = ('open', 'high', 'low', 'close', 'volume')
+    DEFAULT_MINUTELY_SID_CACHE_SIZES = {
+        'close': 3000,
```

Comment:
are we sure we are okay with doubling this?

Reply:
I believe so.  https://github.com/quantopian/zipline/pull/2108 shrinks the expected size of the volume cache significantly for most algorithms, so we have some wiggle room.  For Quantopian, even if we were fully saturating all the caches, the extra memory cost of increasing just this field isn't that big of a deal, and the performance cost of thrashing this cache is bad enough that I think any zipline user who is doing so would almost certainly prefer the extra memory cost here in exchange for the substantil (~50% or more in local testing) speedup this provides.